padle/padle.py

```diff
@@ -0,0 +1,537 @@
+        if(not await self.config.user(ctx.author).start()):
+            return await ctx.send("You have not started the game of PADle yet, try `{}padle start`!".format(prefix))
+        if(await self.config.user(ctx.author).done()):
+            return await ctx.send("You have already played today's PADle!")
+        confirmation = await get_user_confirmation(ctx, "Are you sure you would like to give up?", timeout=20)
+        dbcog = await self.get_dbcog()
+        if not confirmation:
+            if confirmation is None:
+                await send_cancellation_message(ctx, "Confirmation timeout.")
+            return
+        monster = await dbcog.find_monster(str(await self.config.padle_today()), ctx.author.id)
+        await self.config.user(ctx.author).done.set(True)
+        m_embed = EmbedView(EmbedMain(title=MonsterHeader.menu_title(monster).to_markdown(),
+                                      description="PADle #{}".format(await self.config.num_days()),
+                                      url=MonsterLink.ilmina(monster)),
+                            embed_thumbnail=EmbedThumbnail(MonsterImage.icon(monster.monster_id))).to_embed()
+        await ctx.send(embed=m_embed)
+        await ctx.send("Use `{}padle score` to share your score!".format(prefix))
+        async with self.config.user(ctx.author).score() as scores:
+            scores.append(":x:")
+        async with self.config.all_scores() as all_scores:
+            all_scores.append("X")
+
+    @padle.command()
+    async def score(self, ctx):
+        """Share your PADle score"""
+        prefix = (await self.bot.get_prefix(ctx.message))[0]
+        if(not await self.config.user(ctx.author).done()):
+            await ctx.send("You have not done today's PADle yet!")
+            return
+        score = await self.config.user(ctx.author).score()
+        value = "X" if ":x:" in score else len(await self.config.user(ctx.author).guesses())
+        if(ctx.guild is None):
+            msg = "PADle #{}: {}".format(await self.config.num_days(), value) + "\n" + "".join(score)
+            await ctx.send(msg)
+            await ctx.send("*Hint: You can use the command `{}padle score` anywhere "
+                            "and have Tsubaki automatically share your score for you!*".format(prefix))
+        else:
+            await ctx.message.delete()
+            msg = ctx.author.mention + "'s PADle #{}: {}".format(await self.config.num_days(), value) +"\n" + "".join(score)
+            await ctx.send(msg)
+            
+
+    async def generate_padle(self):
+        async def is_day_change():
+            cur_day = datetime.datetime.now().day
+            old_day = await self.config.stored_day()
+            if cur_day != old_day:
+                await self.config.stored_day.set(cur_day)
+                return True
+
+        await self.bot.wait_until_ready()
+        # idk what x is but it wants it to be there
```

Comment:
It's a dummy variable, and I'm pretty sure it's always None.  I'd name it `_` to signify that it has no meaning whatsoever and only exists so that `conditional_iterator` works with `async for`